### PR TITLE
Use Saucelabs package

### PR DESCRIPTION
**Pull request by vrunoa**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,7 +39,7 @@
     "portfinder": "~1.0.10",
     "request": "~2.87.0",
     "requestretry": "~2.0.2",
-    "sauce-connect-launcher": "github:saucelabs/sauce-connect-launcher#v1.3.1",
+    "saucelabs": "^4.6.2",
```
Comment (loyalchow) [inline package.json:42]: To be safer, can this be changed to `~`?

File: src/tunnel.js [hunk 2]
```diff
@@ -1,35 +1,36 @@
-var ngrokLauncher = require('screener-ngrok');
-var Promise = require('bluebird');
-var url = require('url');
-var sauceConnectLauncher = require('sauce-connect-launcher');
-var sauceConnection;
+const ngrokLauncher = require('screener-ngrok');
+const Promise = require('bluebird');
+const url = require('url');
+const Saucelabs = require('saucelabs').default;
```
Comment (loyalchow) [inline src/tunnel.js:4]: Question: Sauce Connect used to be pinned to a specific version (4.6.2 currently). Does `saucelabs` package pin it? If not, can it (or should it) be pinned? 

We currently have the specific version listed in the README and Screener Docs - these docs will need to be updated or removed

File: src/tunnel.js [hunk 3]
```diff
@@ -1,35 +1,36 @@
...
+
+let sauceConnection;
 
 exports.connect = function({ ngrok, sauce }, tries = 0) {
   if (sauce) {
     return new Promise(function(resolve, reject) {
```
Comment (loyalchow) [inline src/tunnel.js:10]: As `saucelabs` package returns a promise, can `new Promise` be removed?

**Review by loyalchow — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (luisVelasquezSauce) [reply package.json]: Updated

File: src/tunnel.js (hunk 2 above)
Comment (luisVelasquezSauce) [reply src/tunnel.js:4]: Default is now 4.6.2 added environment variable SAUCE_CONNECT_VERSION to be able to configure

File: src/tunnel.js (hunk 3 above)
Comment (luisVelasquezSauce) [reply src/tunnel.js]: Done

**Review by loyalchow — APPROVED** [review]

**Pull request merged**